libmamba/CMakeLists.txt

```
@@ -539,7 +539,7 @@ macro(libmamba_create_target target_name linkage output_name)
             set(CMAKE_PREFIX_PATH "$ENV{VCPKG_ROOT}/installed/x64-windows-static-md/")
 
             # For Windows we have a vcpkg based build system right now.
-            find_package(LibArchive REQUIRED)
+            find_package(LibArchive CONFIG REQUIRED)
```

```suggestion
            find_package(LibArchive REQUIRED)
```

---

Ah, my mistake: https://cmake.org/cmake/help/latest/module/FindLibArchive.html

```suggestion
            find_package(LibArchive MODULE REQUIRED)
```